### PR TITLE
Module 1: Add contact form and finish mobile version 

### PR DESCRIPTION
**Pull request by oluwatobi-fasasi**
- Add contact form section.
- Add footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,179 +1,253 @@
...
-                <div class="social">
-                  <a href="https://stackoverflow.com"><img src="images/ic_stackoverflow.svg" alt="stackoverflow"></a>
-                  <a href="https://github.com/"><img src="images/ic_github.svg" alt="github"></a>
-                  <a href="https://twitter.com"><img src="images/ic_twitter.svg" alt="twitter"></a>
-                  <a href="https://linkedin.com"><img src="images/ic_linkedin.svg" alt="linkedin"></a>
-                  <a href="https://medium.com"><img src="images/ic_medium.svg" alt="medium"></a>
-                  <a href="https://angellist.com"><img src="images/ic_angellist.svg" alt="angellist"></a>
-                </div>
-              </footer>
+      <header>
```
Comment (Ibizugbe) [inline index.html:12]: [OPTIONAL] I can see you have put a lot of effort into building your project thus far and your project looks great👍 but let's make some semantic improvements😉😉

A simple page structure for a semantic webpage can take this form😉

```
      <header>
         <-- content -->
      </header>
       
       <main>
         <-- contents-->
       </main>
       
       <footer>
         <-- content -->
       </footer>
```

I suggest your page be divided into the `header` `main` and `footer` sections 😉 Your navigation and logo can occupy the `header` section 👍 The contents of your webpage can occupy the `main` section and the footer should be in the `footer` section😉 using this simple structure will make your webpage more semantic. which will increase your accessibility score and make screen readers navigate your webpage easily😎

File: index.html [hunk 2]
```diff
@@ -1,179 +1,253 @@
...
+          <label for="msg"></label>
+          <br /><br /><br />
+          <textarea
+            id="msg"
+            name="msg"
+            maxlength="500"
+            placeholder=" Enter your message here..."
+            required
+          ></textarea>
+          <button type="submit">Start collaboration</button>
```
Comment (Ibizugbe) [inline index.html:220]: Kindly add the right button state, just as requested in the design😉😉 take note of the image below😎

button states

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi Team 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are passing👍
- [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html)👍.
- [ ] All input fields are wrapped in a single `form` element👍.
- [ ] All input fields have the correct `type` attribute👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_check comment under review😉_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- _check comment under review😉_


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi Team 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] While you did a wonderful job styling your project :+1: It appears that the layout of your mobile version is not responsive and remains static, regardless of the screen size. This could cause issues for users who access your website on smaller mobile devices or tablets, as the layout may not properly adapt to their screen size and resolution. As a project requirement, the mobile version should be responsive until the desktop screen size.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**